microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,257 @@
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"platform":` must be `"edge"` or `"android"`.
+
+* `"attestations":` must include `"attribution_reporting_api"`, `"shared_storage_api"`, `"private_aggregation_api"`, and `"ad_selection_api"`.
+
+   Each `"attestations":` entry must have a single field, `"ServiceNotUsedForIdentifyingUserAcrossSites":`, with either a `true` or `false` value.  `true` means that this service is not used for identifying the user across sites.  `false` means that this service is used for identifying the user across sites.
+
+* `"ownership_token":` is the OT token generated for your individual domain registration.
+
+
+<!-- ====================================================================== -->
+## Deploy the Ad Selection API services
+
+The Ad Selection API uses a trusted execution environment (TEE) to provide a level of assurance for data integrity, data confidentiality, and code integrity; see [Confidential Computing: Hardware-Based Trusted Execution for Applications and Data](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/confidentialcomputing_outreach_whitepaper-8-5x11-1.pdf).  Services provided by the Ad Selection API must run in a TEE to secure the data used by these services.
+
+Ad Selection Services running in a TEE should be deployed on a cloud platform that supports the necessary security features.  Initially, services can be deployed in Azure using [Confidential ACI containers](/azure/container-instances/container-instances-confidential-overview).
+
+The Ad Selection API provides different services that need to be deployed by sellers or buyers.
```

Comment:
```suggestion
The Ad Selection API provides different services that need to be deployed by sellers (SSPs) and buyers (DSPs).
```
instead of defining acro's twice at top of article, an option is to reiterate at first usage in each h2 section, such as here.